src/cap-description.adoc

```diff
@@ -119,10 +119,10 @@ The encoding in xref:cap_perms_encoding32[xrefstyle=short] is chosen to
 facilitate hardware implementations. Therefore, it can be worked out if the
 permissions are granted as follows:
 
-* <<c_perm>>: bit 0 is set
+* <<r_perm>>: bits 3 or 2 are set
```

Comment:
These are the same as before now; I just reordered it so it matches the table columns and the RV64/CAndPerm order.